docs/painless/painless-contexts/painless-context-examples.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[painless-context-examples]]
+=== Context examples
+
+Use the commands below to setup an Elasticsearch cluster to execute the example
+scripts for each context where an example is provided. Each example must be
+executed in the order of the table in the
+<<painless-contexts, previous section>> to work correctly.
+
+
+. Download the
+https://download.elastic.co/demos/painless/contexts/seats.json[seat data]
+for use in the examples. The data is a fictional set of plays and their
+locations and times where each individual document represents a single seat in
+a theater for a single play.
++
+Each document contains the following values:
++
```

Comment:
We generally talk about the "fields" in a document, rather than "values".

Reply:
Fixed.